lib/results_validators/persons_validator.rb

```diff
@@ -46,17 +46,17 @@ def self.dob_validations(dob, competition_id = nil, name = nil)
 
       # Check if DOB is January 1
       if dob.month == 1 && dob.day == 1
-        validation_issues << ValidationWarning.new(:persons, competition_id, DOB_0101_WARNING, name: name)
+        validation_issues << ValidationWarning.new(:dob_jan_one, :persons, competition_id, DOB_0101_WARNING, name: name)
```

Comment:
@gregorbg This is my initial plan for adding IDs. I think it will be better if the IDs are fetched from an enum-like object instead of directly adding the ID here. What do you think?

Reply:
Well, what exactly are you suggesting? I mean, the idea sounds nice, so what kind of "enum-like" structure are you thinking about?

Reply:
I've implemented that - I feel that is more easier than typing. Can you please check the implementation now? Can I go ahead with the same for all definitions or do you have any feedbacks before making change in other definitions?